tools/SetupFlow/DevHome.SetupFlow/Controls/PackageDetailsSettingsCard.xaml

```diff
@@ -0,0 +1,30 @@
+<!-- Copyright (c) Microsoft Corporation and Contributors. -->
+<!-- Licensed under the MIT License. -->
+
+<ctControls:SettingsCard

```

Comment:
All packages settings card controls will be using this control which overwrites certain resources and styles to align with design